docs/data/joy/customization/using-css-variables/using-css-variables.md

```diff
@@ -1,81 +1,96 @@
 # Using CSS variables
 
-<p class="description">Learn how to use CSS variables to customize Joy UI's components.</p>
+<p class="description">Learn how to use CSS variables to customize Joy UI components.</p>
 
-## Theme object
+## Introduction
 
-The `CssVarsProvider` reads the theme input (or the default theme) and creates the CSS variables according to the theme structure.
-It also creates an object that refers to the generated CSS variables under `theme.vars` so that you can use them from the JavaScript theme object.
+To use CSS variables, you must wrap your app with the `<CSSVarsProvider />` utility.
 
-The `theme.vars` is available in all styling APIs that Joy UI offers:
+```jsx
+import { CssVarsProvider } from '@mui/joy/styles';
+
+function App() {
+  return <CssVarsProvider>...</CssVarsProvider>;
+}
+```
 
-1. The `styled` function
+Then you can apply styles based on CSS variables using the `theme.vars.*` notation.
+This notation is available to all styling APIs that Joy UI supports, including the `styled()` function and the `sx` prop.
+
+## Styling APIs
+
+Use the `theme.vars.*` notation with any styling APIs supported by Joy UI:
+
+### styled function
```

Comment:
Maybe we should links to the relevant docs for each of these sections? This page assumes that the reader knows what all of these are, which may not be the case.

Reply:
Added a success callout pointing to the Approaches page — what do you think?

(We might need to do a quick revision on that one, too, to ensure it covers all the styling APIs adequately, as well with demos for them... but for another PR!)